frontend/src/Editor/Inspector/EventManager.jsx

```diff
@@ -246,16 +288,26 @@ export const EventManager = ({
               <div className="row">
                 <div className="col-3 p-2">Modal</div>
                 <div className="col-9">
-                  <SelectSearch
+                  <Select
                     className={`${darkMode ? 'select-search-dark' : 'select-search'}`}
                     options={getComponentOptions('Modal')}
                     value={event.modal?.id ?? event.modal}
                     search={true}
                     onChange={(value) => {
                       handlerChanged(index, 'modal', value);
                     }}
-                    filterOptions={fuzzySearch}
                     placeholder="Select.."
+                    styles={{
+                      ...styles,
+                      menuPortal: (provided) => ({ ...provided, zIndex: 9999 }),
+                      menuList: (base) => ({
+                        ...base,
+                        '::-webkit-scrollbar': {
+                          width: '2px',
+                        },
+                      }),
+                    }}
+                    useMenuPortal="true"
```

Comment:
```suggestion
                    useMenuPortal={true}
```

Reply:
@kavinvenkatachalam Updated the required changes